ui/src/common/DateTimeSelector.tsx

```diff
@@ -9,22 +9,24 @@ interface DateTimeSelectorProps {
     showDate: boolean;
```

Comment:
The time span page looks a little out of place or rather the way the elements are ordered looks buggy as there is no real structure.

Currently, only ~3 time spans are visible on a phone screen, I think this is is not enough. Maybe something like this?

![image](https://github.com/user-attachments/assets/d19115b0-ba91-4711-b1fb-7a56d742959b)



Maybe we could also hide the calender icons and open the date-time from when clicking on the date-time input (disallowing keyboard input)

Reply:
The duration has always 'looked' off, but it is centered. It's just the calendar icon that is low. I also thought they looked a bit off so had it raised up, but opted to just leave it be. 

Default:
![image](https://github.com/user-attachments/assets/32672927-9bf1-4294-b243-70c5c3482048)

Raising the calendar:
![image](https://github.com/user-attachments/assets/728ef8b1-a4bc-4823-801f-aad467e23c05)

Default:
![image](https://github.com/user-attachments/assets/cc48223b-c133-490e-b232-408d4ce35bb7)

Raising the calendar:
![image](https://github.com/user-attachments/assets/6514013f-a85c-4147-8f35-3e978591dac2)

Could also put less gap between the text, though it is slightly off. I think also the calendar button is kind of far off from the textbox, but this distance is to work for longest length local I could find when I changed it. I wonder if it's possible to have the calendar in front of the textbox. Since that would fix that problem.

![image](https://github.com/user-attachments/assets/9f8a2c58-b03e-480a-9364-460a36c32b88)


I definitely want to keep the keyboard input for the time, it is an accessibility feature.


We could probably have the open notes button in the context menu of the 3 dots. I haven't made use of notes yet so this button is mostly wasted space for me, and I woudln't mind the extra click of opening the menu.

Moving the 3 dots into the top right could work, but it would cover up the tags area, or add a right side margin depending on how it's done.

Speaking of the 3 dots menu, there is no confirmation on the delete action. Which would probably be a good idea, especially if we move the open notes action into that.

Reply:
Using some hacky css I've moved the ordering around:
![image](https://github.com/user-attachments/assets/ef800b8a-4925-4973-b9b4-78b2677e1312)

![image](https://github.com/user-attachments/assets/ae2e004f-df92-49df-a99c-27cd738f768f)

Reply:
@jmattheis what are your thoughts on this?

Reply:
Sorry, I'm currently busy, so It'll take some time to respond.

- The calender icon at the start looks better.
- THe duration can stay as it

> I definitely want to keep the keyboard input for the time, it is an accessibility feature.

Oke.

> We could probably have the open notes button in the context menu of the 3 dots. I haven't made use of notes yet so this button is mostly wasted space for me, and I woudln't mind the extra click of opening the menu.

Okay with me.

> Speaking of the 3 dots menu, there is no confirmation on the delete action. Which would probably be a good idea, especially if we move the open notes action into that.

I'm not so sure, recovering this shouldn't be that much of a problem, as there is normally not that much data on a time span.
Maybe something like an undo for the last deleted entry is better, but this should be done in a separate PR.

> Moving the 3 dots into the top right could work, but it would cover up the tags area, or add a right side margin depending on how it's done.

Yeah, but I'd say this is the standard place for a three-dot thingy. upper right corner. Maybe we could reorder the time selection to the top in mobile?

![image](https://github.com/user-attachments/assets/d1c59c74-1f6a-4755-a0e3-054377b531f2)

Reply:
I opted for adding a margin for the 3 dots in the top corner.

Here's what we got now:

https://github.com/user-attachments/assets/a2fb294b-dcdc-4a73-910d-e7ad53213629

I also changed the time dialog to be a dialog instead of a floating thing.

Reply:
Looks much better now :+1: